### PR TITLE
Plehanova homework task2 has been done

### PR DESCRIPTION
**Pull request by Littledrop777**

### CODE REVIEW

File: homework/english/plehanova_Anastasia/lecture1/task2.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+13. to install
+ I need to install Linux for more comfortable working.
+14. to plug in
+  Where can i plug in my phone?
+15. to restore
+ We can restore our files in github.
+16. to uninstall
+  Uninstall this programm if you need no more.
+17. to update
+  Every day in my computure updates different programms.
```
Comment (VolhaZh) [inline homework/english/plehanova_Anastasia/lecture1/task2.md:37]: computEr

File: homework/english/plehanova_Anastasia/lecture1/task2.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Домашнее задание по лексике
+### Напишите по 1 предложению к каждому слову из лексического блока в 1 уроке (слова есть в презентации).
+
+1. to boot
+  I need to boot OS after instaling some updates.
```
Comment (VolhaZh) [inline homework/english/plehanova_Anastasia/lecture1/task2.md:5]: instaLLing

**Review by VolhaZh — CHANGES_REQUESTED** [review]
find just little mistakes need to be corrected
Lgfm

**Review by ShelestV — DISMISSED** [review]
lgfm

File: homework/english/plehanova_Anastasia/lecture1/task2.md (hunk 1 above)
Comment (Littledrop777) [reply homework/english/plehanova_Anastasia/lecture1/task2.md:37]: change

**Review by breackinumber — APPROVED** [review]
Gj

**Review by Alex99905 — APPROVED** [review]

**Pull request merged**